### PR TITLE
Migrated test_replica_eager_lock

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated
[test_replica_eager_lock](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/afr/test_replica_eager_lock.py)

Updates: #292

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

**Comment by aujjwal-redhat** [comment]
This tc is complete.
Issue :
Sometimes the last part of the tc `gluster v profile test_replica_eager_lock-rep info | grep OPENDIR | awk '{print$8}`

```log
[2021-07-16 16:35:14,898] INFO  - Running gluster v profile test_replica_eager_lock-rep info | grep OPENDIR | awk '{print$8}' on 10.70.43.115
[2021-07-16 16:35:15,193] DEBUG  - {'msg': ['1\n', '5\n'], 'Flag': True, 'node': '10.70.43.115', 'cmd': "gluster v profile test_replica_eager_lock-rep info | grep OPENDIR | awk '{print$8}'", 'error_code': 0}

```
While sometimes:
```log
[2021-07-16 16:35:55,817] INFO  - Running gluster v profile test_replica_eager_lock-dist-rep info | grep OPENDIR | awk '{print$8}' on 10.70.43.115
[2021-07-16 16:35:56,085] DEBUG  - {'msg': ['1\n', '1\n'], 'Flag': True, 'node': '10.70.43.115', 'cmd': "gluster v profile test_replica_eager_lock-dist-rep info | grep OPENDIR | awk '{print$8}'", 'error_code': 0}
```

Because of this sometimes tc fails.

**Comment by aujjwal-redhat** [comment]
```sh

Results for afr
Test nature : disruptive
TestName : test_replica_eager_lock
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     rep     |     PASS    |        00:00:38       |      NA     |
|   dist-rep  |     PASS    |        00:00:46       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   1   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 01 minutes 34 seconds
```

File: tests/functional/afr/test_replica_eager_lock.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        for mount_obj in self.mounts:
+            proc = redant.create_deep_dirs_with_files(mount_obj['mountpath'],
+                                                      self.counter,
+                                                      2, 15, 5, 25,
+                                                      mount_obj['client'])
+            self.all_mounts_procs.append(proc)
+            self.counter = self.counter + 10
+
+        # this command should not get hang while io is in progress
+        for i in range(20):
```
Comment (srijan-sivakumar) [inline tests/functional/afr/test_replica_eager_lock.py:105]: seeing that i isn't even used, we can replace `i` with `_`.

**Review by srijan-sivakumar — CHANGES_REQUESTED** [review]

File: tests/functional/afr/test_replica_eager_lock.py (hunk 1 above)
Comment (aujjwal-redhat) [reply tests/functional/afr/test_replica_eager_lock.py:105]: ack

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**